client/sites/components/site-preview-pane/preview-pane-header-buttons.tsx

```diff
@@ -62,6 +65,8 @@ const PreviewPaneHeaderButtons = ( { focusRef, itemData }: Props ) => {
 					environment={ environment }
 					productionSiteId={ productionSiteId }
 					stagingSiteId={ stagingSiteId }
+					isSyncInProgress={ isSyncInProgress }
+					onSyncStart={ () => resetSyncStatus() }
```

Comment:
Very minor nitpick: We can pass only `resetSyncStatus` here so we can save some characters 😁